docs/source/support/faq.rst

```diff
@@ -16,10 +16,10 @@ The answers in the FAQ regularly reference the network characters "**Alice**," "
 General
 -------
 
-Q: What is the network name for Incentivized Testnet?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Q: What is the network name for NuCypher's testnet?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The network name for incentivized testnet is ``gemini``.
+The network name for testnet is ``gemini``.
```

Comment:
This will change soon, but for the moment is OK.

Reply:
Needs an Issue to track IMO.

Reply:
Filed #2057 .